README.md

```diff
@@ -152,6 +153,19 @@ Mbed TLS, like Mbed Crypto, supports being built as a CMake subproject. One can
 use `add_subdirectory()` from a parent CMake project to include Mbed TLS as a
 subproject.
 
+### VCPKG
+
+You can download and install mbedtls using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install mbedtls
+
+The mbedtls port in vcpkg is kept up to date by Microsoft team members and community contributors.
+If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
```

Comment:
Mbed TLS is also a Debian package, a FreeBSD port, and Arch Linux package, etc.. We would need a long list to describe all the different package managers and their idiosyncrasies if we wanted to describe all the ways to get and install. In practice, the ways are too plentiful and varied to make it worthwhile to maintain that information within this library. We shouldn't try to duplicate documentation for how to use package managers here.

This sort of information is best left to the package manager's documentation, or for Mbed TLS specifically, a blog post or howto article, not in the README that describe how to build the library.

Reply:
@Patater, thanks for your reviewing!

How about updating it to 'Alternatively, If you are using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager you can download and install mbedtls with CMake integration in a single command:  vcpkg install mbedtls'?

Reply:
You would be able to download and build/install using FreeBSD ports or Debian source packages as well, but these are out of scope in our README and we don't mention them.

Until we have a better place to put this sort of tangential information, like a wiki or guest blog, we can't accept this sort of contribution. Sorry.